doc/lua_api.txt

```diff
@@ -6223,6 +6223,10 @@ Player properties need to be saved manually.
 
         automatic_rotate = 0,
         -- Set constant rotation in radians per second, positive or negative.
+        -- Object rotates along the local Y-axis, and works with set_rotation.
+        -- This can be used to effectively change the axis of automatic_rotate
+           to a mod's requirement.
+        -- Can be used to simulate propellers, fans, etc.
```

Comment:
IMO this is superfluous. Modders will find use-cases for this parameter fast enough.

Reply:
Done